lib/file_helper.rb

```diff
@@ -52,7 +52,7 @@ def self.download(
     retain_on_max_file_size_exceeded: false
   )
     url = "https:" + url if url.start_with?("//")
-    raise Discourse::InvalidParameters.new(:url) unless url =~ %r{\Ahttps?://}
+    # raise Discourse::InvalidParameters.new(:url) unless url =~ %r{\Ahttps?://}
```

Comment:
TODO: Need a better way of doing this obviously; we want to be able to download from http

Reply:
Maybe in Rails.env.development/Rails.env.test?